Sources/SourceKitLSP/Clang/ClangLanguageService.swift

```diff
@@ -621,8 +621,8 @@ extension ClangLanguageService {
     }
     return try await forwardRequestToClangd(req)
   }
-
-  func openGeneratedInterface(_ request: OpenGeneratedInterfaceRequest) async throws -> GeneratedInterfaceDetails? {
+    
```

Comment:
Could you run swift-format on your changes? https://github.com/swiftlang/sourcekit-lsp/blob/main/CONTRIBUTING.md#formatting